frontend/packages/design-system/src/types/icons.ts

```diff
@@ -1,14 +1,8 @@
 export type IconProps = {
   name: IconName;
-  variant: IconVariants;
+  color: string;
 };
 
-export enum IconVariants {
-  DARK = '#000000',
-  LIGHT = '#FFFFFF',
-  LINK = '#1E4A7D',
-}
-
 export enum IconName {
```

Comment:
Do we need this `enum`? If the `name` prop is `keyof typeof iconMap` then I'm not sure this is necessary and it is difficult to mantain.

Reply:
yeah, this is currently only used for the add ons in storybook, might be replaceable

Reply:
but then we would have to export iconMap, not sure if we want that

Reply:
Yeah I'd probably replace it with something like `keyof typeof iconMap`. That way we don't have to remember to add it here when adding new icons.